docs-website/src/components/MarkpromptHelp/index.jsx

```diff
@@ -1,53 +1,207 @@
+    </Markprompt.Root>
+  );
+};
+
+async function sendFeedback(data = {}) {
+  const response = await fetch("https://hlmqfkovdugtwoddrtkt.supabase.co/rest/v1/queries", {
+    method: "POST",
+    headers: {
+      "Content-Type": "application/json",
+      Prefer: "return=minimal",
+      apikey:
+        "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6ImhsbXFma292ZHVndHdvZGRydGt0Iiwicm9sZSI6ImFub24iLCJpYXQiOjE2ODI5NzQwMjQsImV4cCI6MTk5ODU1MDAyNH0.wNggN7IK43j6KhJmFlELo8HmkTBIQwWt-Rx8vOYXBI8",
+    },
+    redirect: "follow",
+    referrerPolicy: "no-referrer",
+    body: JSON.stringify(data),
+  });
+  return response;
+}
+
+const Feedback = () => {
+  const { state, prompt } = useContext(Markprompt.Context);
+  const [feedbackSubmitted, setFeedbackSubmitted] = useState(false);
+
+  useEffect(() => {
+    setFeedbackSubmitted(false);
+    if (state === "preload") {
+      sendFeedback({
+        query: prompt,
+      }).then((response) => {
+        console.log(response.ok ? "Logged query" : "Error logging query");
+      });
+    }
+  }, [state]);
+
+  const handleClick = (rating) => {
+    sendFeedback({
+      query: prompt,
+      result_rating: rating,
+    }).then((response) => {
+      setFeedbackSubmitted(true);
+      console.log(response.ok ? "Logged feedback" : "Error logging feedback");
+    });
+  };
 
   return (
-    <div>
-      <button className={clsx(styles.markpromptButton, "button button--primary shadow--tl")} onClick={showModal}>
-        <BotIcon />
-      </button>
+    <div className={styles.feedback} data-loading-state={state}>
+      {feedbackSubmitted ? (
+        <>Thanks for your feedback!</>
+      ) : (
+        <>
+          Was this answer helpful?{" "}
+          <div className={styles.feedbackButtons}>
+            <button onClick={() => handleClick("positive")}>
+              <LikeOutlined />
+            </button>
+            <button>
+              <DislikeOutlined onClick={() => handleClick("negative")} />
+            </button>
+          </div>
+        </>
+      )}
     </div>
   );
 };
 
-const MarkpromptHelp = () => (
-  <ModalProvider>
-    <MarkpromptModal />
-  </ModalProvider>
+const Caret = () => {
+  const { answer } = useContext(Markprompt.Context);
+
+  if (answer) {
+    return null;
+  }
+
+  return <span className={styles.caret} />;
+};
+
+const capitalize = (text) => {
+  return text.charAt(0).toUpperCase() + text.slice(1);
+};
+
+const removeFileExtension = (fileName) => {
+  const lastDotIndex = fileName.lastIndexOf(".");
+  if (lastDotIndex === -1) {
+    return fileName;
+  }
+  return fileName.substring(0, lastDotIndex);
+};
+
+const Reference = ({ referenceId, index }) => {
+  return (
+    <li
+      key={referenceId}
+      className={styles.reference}
+      style={{
+        animationDelay: `${100 * index}ms`,
+      }}
+    >
+      <a href={removeFileExtension(referenceId)}>{capitalize(removeFileExtension(referenceId.split("/").slice(-1)[0]))}</a>
```

Comment:
The mapping from `.md` file to path in the website is not always direct

I think you'll need to inspect the routing details using https://docusaurus.io/docs/docusaurus-core#usePluginData to do the mapping more reliably - ideally we also resolve the doc titles from there. 

You can reference the docusuarus debug route http://localhost:3000/__docusaurus/debug/globalData 

Also, these links should be absolute paths and not relative ones

Reply:
This should be working now! 

I couldn't find a way to get the doc titles so am just using a portion of the full path for that, but let me know if you have any ideas for how to grab them.